### ISSUE TITLE
Accessibility: Wrong element had `aria-expanded` removed.

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1989]
In https://github.com/silverstripe/silverstripe-admin/pull/1978 the `aria-expanded` attribute should have been removed from the `nav#cms-menu` but was removed from the `button.cms-help__menu` instead.

## PRs

- https://github.com/silverstripe/silverstripe-admin/pull/1990
